hudi-spark-datasource/hudi-spark/src/main/scala/org/apache/spark/sql/hudi/command/procedures/RollbackToInstantTimeProcedure.scala

```diff
@@ -73,10 +73,14 @@ class RollbackToInstantTimeProcedure extends BaseProcedure with ProcedureBuilder
         throw new HoodieException(s"Commit $instantTime not found in Commits $completedTimeline")
       }
 
-      val result = if (client.rollback(instantTime)) true else false
-      val outputRow = Row(result)
+      val outputRow = new util.ArrayList[Row]
+      val allInstants: List[HoodieInstant] = completedTimeline
+        .findInstantsAfterOrEquals(instantTime, Integer.MAX_VALUE).getReverseOrderedInstants.toArray()
+        .map(r => r.asInstanceOf[HoodieInstant]).toList
 
-      Seq(outputRow)
+      allInstants.foreach(p => outputRow.add(Row(client.rollback(p.getTimestamp), p.getTimestamp)))
```

Comment:
why we need outputRow， 
how about return allInstants.map(p => Row(client.rollback(p.getTimestamp), p.getTimestamp))  directly

Reply:
The request argument of `rollback_to_instant` has the `instant_time`, therefore the return result doesn't need the `instant_time`.

Reply:
> why we need outputRow， how about return allInstants.map(p => Row(client.rollback(p.getTimestamp), p.getTimestamp)) directly

A bit hesitant to add `limit`